sw/device/silicon_creator/lib/manifest.h

```diff
@@ -444,11 +447,14 @@ inline epmp_region_t manifest_code_region_get(const manifest_t *manifest) {
  * @param manfiest A manifest.
  * return Entry point address.
  */
+OT_WARN_UNUSED_RESULT
 inline uintptr_t manifest_entry_point_get(const manifest_t *manifest) {
   return (uintptr_t)manifest + manifest->entry_point;
 }
 
 #define DEFINE_GETTER(index_, type_, name_, id_, _)                            \
+  OT_WARN_UNUSED_RESULT                                                        \
+                                                                               \
```

Comment:
Nit: extra newline on 457.

Reply:
good catch but clang-format did it, and I didn't add clang-format off because it formats the rest properly.